cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -85,7 +85,7 @@
       cronjob at:'30 14 * * *', do:dashboard_dir('bin','scheduled_pd_workshop_emails')
       cronjob at:'* 14 * * 1', do:dashboard_dir('bin','scheduled_pd_application_emails')
       cronjob at:'10 */12 * * *', do:dashboard_dir('bin','refresh_pd_workshop_material_orders')
-      cronjob at:'*/1 * * * *', do:dashboard_dir('bin', 'cron', 'sync_jotforms')
+      cronjob at:'*/1 * * * *', do:dashboard_dir('bin', 'sync_jotforms')
```

Comment:
It should have been either `deploy_dir('bin', 'cron', ...` or `dashboard_dir('bin', ...)` but instead was an erroneous mix.

Reply:
ohhh I should have noticed that.